app/models/ticket.rb

```diff
@@ -1,26 +1,28 @@
 class Ticket < ApplicationRecord
   belongs_to :ticket_desk
   belongs_to :cinema_hall
-  belongs_to :client
+  belongs_to :movie
+  
   validate :id, :if => :exist?
-  validates :client_id, numericality: { only_integer: true }
   validates :ticket_desk_id, numericality: { only_integer: true }
   validates :cinema_hall_id, numericality: { only_integer: true }
+  validates :movie_id, numericality: { only_integer: true }
   validates :paid, presence: true, inclusion: { in: [true, false] }
 
-
-  def exist?
-      Ticket.exists?(id: self.ticket_id)
-  end
-
-  def ticket_id
-    self.id
-  end
-
-  def ticket_paid
-    self.paid
-  end
-
-
   
-end
+    def exist?
+        Ticket.exists?(id: self.ticket_id)
+    end
```

Comment:
What's the purpose of this method? If you want to check if something is saved in the database you have methods like `persisted?`, e.g.:

```ruby
ticket = Ticket.new
ticket.persisted? #=> false
ticket.save
ticket.persisted? #=> true
ticket = Ticket.find(id)
ticket.persisted? #=> true
```